src/pages/welcomePage.js

```diff
@@ -1,19 +1,73 @@
-import { USER_INTERFACE_ID, START_QUIZ_BUTTON_ID } from '../constants.js';
+import { USER_INTERFACE_ID, START_QUIZ_BUTTON_ID , CONFIRM_BUTTON ,INPUT_NAME , INPUT_P_TAG , WELCOME_MESSAGE  } from '../constants.js';
 import { createWelcomeElement } from '../views/welcomeView.js';
 import { initQuestionPage } from './questionPage.js';
 
 export const initWelcomePage = () => {
   const userInterface = document.getElementById(USER_INTERFACE_ID);
   userInterface.innerHTML = '';
+  
 
   const welcomeElement = createWelcomeElement();
   userInterface.appendChild(welcomeElement);
 
   document
     .getElementById(START_QUIZ_BUTTON_ID)
     .addEventListener('click', startQuiz);
-};
 
+    // we need add text in the input and when clicked confirm button can see user name and can start the quiz
+    // Also we need confirm btn and player name and welcome message 
+    const confirmButton = document.getElementById(CONFIRM_BUTTON);
+    const startButton = document.getElementById(START_QUIZ_BUTTON_ID);
+    const nameInput = document.getElementById(INPUT_NAME);
+    const namePrompt = document.getElementById(INPUT_P_TAG);
+    const welcomeMessage = document.getElementById(WELCOME_MESSAGE);
+  
+ 
+  function updateButtonState() {
+    nameInput.value = nameInput.value.trimStart(); 
+    const name = nameInput.value;
+    if (name) {
+      confirmButton.classList.remove('button-disabled');
```

Comment:
- You don't need to use extra classes to style the disabled button. Don't add "button-disabled" or "button-enabled" classes to buttons.

In css use 
```
#confirm-button:disabled {
 ...
}
```
You can add styling to the disabled button like this. 
In order to style enabled version of the button. You will assign the necessary stylings to #confirm-button in css.